index.html

```diff
@@ -55,36 +56,44 @@ <h1 class="h1-title">Colors</h1>
                     <br>#0D4781</p>
             </div>
         </div>
+		<div id="" class="code--info--separator"></div>
         <div class="f-row">
             <div class="f-item" style="margin-top:40px;">
                 <p>
-                    <h1 class="h1-title">Title</h1></p>
+                    <h2>Headings</h2></p>
             </div>
         </div>
         <div class="f-row">
             <div class="f-item">
-                <h1>Header 1</h1>
-                <h2>Header 2</h2>
-                <h3>Header 3</h3>
+                <h1>Heading 1</h1>
+                <h2>Heading 2</h2>
+                <h3>Heading 3</h3>
+				<h4>Heading 4</h4>
+                <h5>Heading 5</h5>
```

Comment:
6 levels of headers seems far too much. There are only 3 in style.binary.com. Please stick to 3. What problems does it cause with Manros' design? Pls ask Manros to simplify his design.